cmd/check.go

```diff
@@ -55,9 +57,14 @@ Flags:
 		}
 
 		if checkflags.ListPackages {
+			headings := "Package Name\tCatalog Source\tCatalog Source Namespace"
+			w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
```

Comment:
For future possibility of testing this, I would use cobra.Command.OutOrStdout()

Reply:
+1 for testing possibility.